src/thriveopedia/fossilisation/FossilisedSpecies.cs

```diff
@@ -119,15 +119,24 @@ public static bool IsSpeciesAlreadyFossilised(string name, List<string>? existin
     /// </summary>
     /// <param name="fossilName">The name of the .thrivefossil file (including extension)</param>
     /// <returns>The species saved in the provided file</returns>
```

Comment:
Can you update the `returns` documentation please?

Reply:
Done.